openedx/features/idea/forms.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+from django.core.exceptions import ValidationError
+from django.db import transaction
+from django.forms import CharField, ModelForm, Textarea, TextInput
+from django.utils.translation import ugettext_lazy as _
+
+from lms.djangoapps.onboarding.helpers import COUNTRIES, get_country_iso
+from lms.djangoapps.onboarding.models import Organization
+
+from .constants import COUNTRY_MAX_LENGTH, IDEA_FILE_MAX_SIZE, IDEA_IMAGE_MAX_SIZE, ORGANIZATION_NAME_MAX_LENGTH
+from .helpers import validate_image_dimension, validate_file_size
+from .models import Idea
+
+
+class IdeaCreateForm(ModelForm):
+    """
+    A model form to create new Idea. Separate fields for organization_name and country_name are used
+    to make model field organization and country work with custom widget on frontend.
+    """
+
+    prefix = 'idea'
+    label_suffix = ''
+
+    organization_name = CharField(
+        max_length=ORGANIZATION_NAME_MAX_LENGTH,
+        label=_('Organization Name'),
+        required=True,
+        widget=TextInput(),
+    )
+
+    country_name = CharField(
+        max_length=COUNTRY_MAX_LENGTH,
+        label=_('Country'),
+        required=True,
+        widget=TextInput(),
+    )
+
+    class Meta:
+        model = Idea
+        exclude = ('user', 'favorites', 'organization', 'country',)
+
+        widgets = {
+            'overview': Textarea(),
+        }
+
+    def __init__(self, *args, **kwargs):
+        super(IdeaCreateForm, self).__init__(*args, **kwargs)
+        self.label_suffix = ''
+
+        initial_arguments = kwargs.get('initial', {})
+        self.user = initial_arguments.get('user', None)
+        self.disable_field('organization_name', initial_arguments)
+        self.disable_field('country_name', initial_arguments)
+        self.disable_field('city', initial_arguments)
```

Comment:
You are quoting wrong reference, for select element readonly property does not work see this [link](https://stackoverflow.com/a/7730719). Also as mentioned in SO answer, "A disabled element isn't editable and isn't sent on submit", which alines with the requirement, because these fields are already initialised from backend.
 
Qasim have used extra styling to make it work, also his requirements are different, on marketplace organisation is always disabled.

Reply:
I wasn't sure about it that's why i asked with can we ? instead of asking to change it.